src/ecalc/libraries/libecalc/common/tests/core/models/compressor_modelling/test_fluid.py

```diff
@@ -132,3 +132,34 @@ def test_set_new_pressure_and_enthalpy_or_temperature(fluid_streams: List[List[F
         new_enthalpies_all_fluids_after_setting_new_temperature,
         new_enthalpies_all_fluids,
     )
+
+
+def test_fluid_mixing(dry_fluid, rich_fluid):
+    """Test mixing two fluids together, check that the order does not change"""
+
+    dry_fluid_stream = FluidStream(fluid_model=dry_fluid)
+    rich_fluid_stream = FluidStream(fluid_model=rich_fluid)
+
+    mix_rich_into_dry = dry_fluid_stream.mix_in_stream(
+        other_fluid_stream=rich_fluid_stream,
+        self_mass_rate=1,
```

Comment:
should use something else than 1 for mass rate for streams? too simple?

Reply:
Mass rate is only used to define the mixing of the components. So it is basicly just a fraction. " 1 part of self mixed with 1 part of new fluid", I guess we could change to 2:1 ratio if you think it could catch some faulty logic

Reply:
ok, just checking. should in general have 1 check per test, so that should probably be covered in another test, or might already have been done...?